website/scripts/build-static-content.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
         let queriesWithProblematicContributors = [];
         let queries = YAML.parseAllDocuments(yaml).map((yamlDocument)=>{
           let query = yamlDocument.toJSON().spec;
+          query.kind = yamlDocument.toJSON().kind;
```

Comment:
Does this mean the code related to “purpose” in this file is no longer needed?  If so, let’s remove it now so we catch ‘em all 🔴💥

Reply:
We can! I updated the commented-out conditionals (lines 38 and 41) to work with the updated query spec (purpose -> kind, remediation -> resolution) for when/if we enable them.